### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by SovaV**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "modern-normalize": "^1.1.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router": "^6.2.1",
+    "react-router-dom": "^6.2.1",
```
Comment (Prykhodko-Pasha) [inline package.json]: Please delete the carets

File: src/components/MenuNavigation/MenuNavigation.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <nav>
+      <NavLink
+        to="/"
+        className={({ isActive }) => (isActive ? 'active-style' : 'link')}
+      >
+        <svg width="44" height="44">
+          <use href={`${sprite}#icon-home`}></use>
+        </svg>
+        Главная
```
Comment (Prykhodko-Pasha) [inline src/components/MenuNavigation/MenuNavigation.js]: We've agreed to write everything in English

File: package.json (hunk 1 above)
Comment (SovaV) [reply package.json]: done

File: src/components/MenuNavigation/MenuNavigation.js (hunk 2 above)
Comment (SovaV) [reply src/components/MenuNavigation/MenuNavigation.js]: done

**Review by Prykhodko-Pasha — APPROVED** [review]

**Review by od-frontender — APPROVED** [review]

**Review by od-frontender — APPROVED** [review]

**Pull request closed without merging**